packages/docs/docs/getting-started/media.mdx

```diff
@@ -0,0 +1,179 @@
+  audioOffset: 1.5, // delays the audio by 1.5 second
+});
+```
+
+![](@site/static/img/media/editor-audio-track-delayed.png)
+
+### Supported Formats
+
+A list of supported file formats can be found
+[here](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Audio_codecs#common_codecs)
+
+## Images
+
+Another type of media Motion Canvas supports are images. To use an image in the
+animation we first need to import the image:
+
+```ts
+// my-animation/
+// ├─ images/
+// │  └─ example.png
+// ├─ src/
+// ...
+import examplePng from '../../images/example.png';
+```
+
+Then we can use Motion Canvas [`Image`](/api/2d/components/Image) component to
+display the image as follows:
+
+```tsx
+import {makeScene2D} from '@motion-canvas/2d/lib/scenes';
+import {Image} from '@motion-canvas/2d/lib/components';
+
+import examplePng from '../../images/example.png';
+
+export default makeScene2D(function* (view) {
+  view.add(<Image src={examplePng} />);
+});
+```
+
+Images can also be animated like every other component in Motion Canvas. For
+this we just need to create a [`Reference`](/docs/references) and pass that to
+the [`Image`](/api/2d/components/Image):
+
+```tsx
+// ...
+import {createRef} from '@motion-canvas/core/lib/utils';
+import {all} from '@motion-canvas/core/lib/flow';
+
+export default makeScene2D(function* (view) {
+  const imageRef = createRef<Image>();
+
+  view.add(<Image ref={imageRef} src={examplePng} scale={2} />);
+
+  yield* all(
+    imageRef().scale(2.5, 1.5).to(2, 1.5),
+    imageRef().absoluteRotation(90, 1.5).to(0, 1.5),
+  );
+});
+```
+
+<AnimationPlayer name="media-image" small />
+
+### Supported Formats
+
+A list of supported file formats can be found
+[here](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types)
+
+## Video
+
+Videos in Motion Canvas work similar to images. Firstly we need to import our
+video we want to display:
+
+```ts
+// my-animation/
+// ├─ videos/
+// │  └─ example.mp4
+// ├─ src/
+// ...
+import exampleMp4 from '../../videos/example.mp4';
+```
+
+Then we can use Motion Canvas [`Video`](/api/2d/components/Video) component to
+display the video as follows:
+
+```tsx
+import {makeScene2D} from '@motion-canvas/2d/lib/scenes';
+import {Video} from '@motion-canvas/2d/lib/components';
+
+import exampleMp4 from '../../videos/example.mp4';
+
+export default makeScene2D(function* (view) {
+  view.add(<Video ref={videoRef} src={exampleMp4} />);
+});
+```
+
+However this will only display the video and will not play it. In order to play
+the video we need to create a [`Reference`](/docs/references) to the element.
+
+```tsx
+// ...
+import {createRef} from '@motion-canvas/core/lib/utils';
+
+export default makeScene2D(function* (view) {
+  const videoRef = createRef<Video>();
+
+  view.add(<Video ref={videoRef} src={exampleMp4} />);
+});
+```
+
+Now that we have a [`Reference`](/docs/references) to the
+[`Video`](/api/2d/components/Video) element we can use it to animate the video
```

Comment:
nit: "...to the Video element, we can use it to animate the video, as well as play and pause it." (added some commas)